src/applications/vaos/new-appointment/components/DateTimeSelectPage/index.jsx

```diff
@@ -1,107 +1,184 @@
 import {
   addDays,
   addMonths,
+  isSameDay,
+  isValid,
   lastDayOfMonth,
   parseISO,
   startOfMonth,
 } from 'date-fns';
+import { formatInTimeZone } from 'date-fns-tz';
 import { scrollToFirstError } from 'platform/utilities/scroll';
 import PropTypes from 'prop-types';
 import React, { useEffect, useState } from 'react';
 import { shallowEqual, useDispatch, useSelector } from 'react-redux';
 import { useHistory } from 'react-router-dom';
 
-import InfoAlert from '../../../components/InfoAlert';
-
 import { fetchFutureAppointments } from '../../../appointment-list/redux/actions';
 import {
   getUpcomingAppointmentListInfo,
   selectUpcomingAppointments,
 } from '../../../appointment-list/redux/selectors';
 import CalendarWidget from '../../../components/calendar/CalendarWidget';
 import FormButtons from '../../../components/FormButtons';
+import InfoAlert from '../../../components/InfoAlert';
 import NewTabAnchor from '../../../components/NewTabAnchor';
 import useIsInitialLoad from '../../../hooks/useIsInitialLoad';
-import { getRealFacilityId } from '../../../utils/appointment';
-import { FETCH_STATUS } from '../../../utils/constants';
+import { DATE_FORMATS, FETCH_STATUS } from '../../../utils/constants';
 import { scrollAndFocus } from '../../../utils/scrollAndFocus';
+import { getFormattedTimezoneAbbr } from '../../../utils/timezone';
 import { getPageTitle } from '../../newAppointmentFlow';
 import {
   getAppointmentSlots,
   onCalendarChange,
-  requestAppointmentDateChoice,
   routeToNextAppointmentPage,
   routeToPreviousAppointmentPage,
+  routeToRequestAppointmentPage,
 } from '../../redux/actions';
-import {
-  getChosenClinicInfo,
-  getDateTimeSelect,
-  selectEligibility,
-} from '../../redux/selectors';
-import WaitTimeAlert from './WaitTimeAlert';
+import { getChosenClinicInfo, getDateTimeSelect } from '../../redux/selectors';
+import UrgentCareLinks from '../UrgentCareLinks';
 
 const pageKey = 'selectDateTime';
 
-function renderContent({ dispatch, isRequest, facilityId, history }) {
-  // Display this content when the facility is configured to accept appointment
-  // request
-  if (isRequest) {
+function handleClick(history, dispatch) {
+  return e => {
+    // Stop default behavior for anchor tag since we are using React routing.
+    e.preventDefault();
+    dispatch(routeToRequestAppointmentPage(history, pageKey));
```

Comment:
@simiadebowale I modeled the request link on your PR here: https://github.com/department-of-veterans-affairs/vets-website/pull/38119. The functionality looked find to me when I tested it but let me know if I missed anything.